README.md

```diff
@@ -123,6 +123,7 @@ Spiderable.debug = true
  - Create template which you prefer to return, when page is not found
  - Set `Spiderable.customQuery`
  - Set iron router's `notFoundTemplate`
+ - Include a comment '<!-- response:status-code=404 -->' on your template. This way, we can ensure spiderable sends a 404 status code in the response headers
```

Comment:
@jazeee I'll test it tonight, will let you know if I found any issues. Thank you.